bin/dipy_bet

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env python
+
+from dipy.workflow.segment import median_otsu_bet
+from dipy.fixes import argparse as arg
+
+parser = arg.ArgumentParser()
+parser.add_argument('input', action='store')
+parser.add_argument('output_directory', action='store')
```

Comment:
I would make this argument of the 
'''
--out_dir
'''
type. If it is not provided then the output will be saved in the same directory as the one of the input. 
For example if we had 3 subjects:
```
subj1/t1.nii.gz
subj2/t2.nii.gz
subj3/t3.nii.gz
```
then if the out_dir is not provided the results will go at
```
subj1/t1_bet.nii.gz
subj2/t2_bet.nii.gz
subj3/t3_bet.nii.gz
```
But if it is provided then there are two options
a) A non-complete path is provided e.g. 
```
--out_dir bet_folder
```
then we expect that this folder name will be added in the corresponding existing paths of each subject
subj1/bet_folder/t1_bet.nii.gz
subj2/bet_folder/t2_bet.nii.gz
subj3/bet_folder/t3_bet.nii.gz

b) A complete path is given e.g. 
```
--out_dir /tmp/
```
In this case we can need to do something different. The result should come out as
```
/tmp/subj1/t1_bet.nii.gz
/tmp/subj2/t2_bet.nii.gz
/tmp/subj3/t3_bet.nii.gz
```
Which is great for debugging!
How does that sound?